save-frontend/src/main/kotlin/org/cqfn/save/frontend/components/views/ProjectView.kt

```diff
@@ -564,16 +576,72 @@ class ProjectView : RComponent<ProjectExecutionRouteProps, ProjectViewState>() {
                 })
             }
             // ===================== RIGHT COLUMN ======================================================================
-            div("col-2 ml-2") {
+            div("col-3 ml-2") {
                 div("text-xs text-center font-weight-bold text-primary text-uppercase mb-3") {
                     +"Information"
+                    button(classes = "btn btn-link text-xs text-muted text-left p-1 ml-2") {
+                        +"Edit"
+                        attrs.onClickFunction = {
+                            turnEditMode(off = false)
+                        }
+                    }
                 }
 
                 child(cardComponent {
-                    infoText("Tested tool name: ", project.name)
-                    infoText("Description: ", project.description ?: "")
-                    infoText("Tested tool Url: ", project.url ?: "")
-                    infoText("Test project owner: ", project.owner)
+                    val newProjectInformation: MutableMap<String, String> = mutableMapOf()
+                    form {
+                        projectInformation.putAll(
+                            projectInformation.keys.zip(listOf(project.name, project.description ?: "", project.url ?: "", project.owner))
+                        )
+                        projectInformation
+                            .forEach { (header, text) ->
+                                div("control-group form-inline") {
+                                    label(classes = "control-label col-auto") {
+                                        +header
+                                    }
+                                    div("controls col-auto") {
+                                        input(InputType.text, classes = "form-control-plaintext") {
+                                            attrs.id = header
+                                            attrs.placeholder = text
```

Comment:
I use here the placeholder, since in other way, the representation of all fields on frontend will be `fixed`, it's means, that while you will try to change some field and enter new name, on frontend the value will fixed hold old value

Reply:
What do you mean? If `input` is not `disabled`, then it should be editable disregarding whether data is set as placeholder or value.

Reply:
Yes, I thought so too, but it didn't worked as expected.

I don't understand the real reason, but it looks like this, when I use `value` instead of `placeholder`:

![Capture](https://user-images.githubusercontent.com/26010098/139804326-24b75976-6335-4c2f-9271-0cc59bca1df8.PNG)

The field `Tested tool name` becomes unchangeable during the input from keybord and collect only one symbol, while I enter the `Some value`

Reply:
You can try to use `defaultValue` (see [docs](https://developer.mozilla.org/en-US/docs/Web/API/HTMLInputElement) for `HTMLInputElement`).
Also - if you use `placeholder`, does the user need to type the whole text? I would expect that placeholder disappears, once input gets focus.

Reply:
Seems, `defaultValue` is more appropriate, thanks, it works better, changed

About `placeholder` - user were need to type the whole text only once, after first `save` or `cancel`, it were possible to change text in any place.

With `defaultValue` all's good immediately